### PR TITLE
symfony3 compatibility

### PR DESCRIPTION
**Pull request by siwymilek**
I changed some requirements in composer.json 

| Q               | A
| --------------- | ---
| Bug fix?        | no
| New feature?    | no
| BC breaks?      | no
| Related tickets | #75 
| License         | MIT

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -24,22 +24,22 @@
         }
     ],
     "require": {
-        "php": "^5.6|^7.0",
+        "php": ">=5.5.9",
```
Comment (pamil) [inline composer.json]: Should be reverted, we cannot guarantee it will work correctly on PHP 8.0.

Comment (siwymilek) [inline composer.json]: you're right, fixed

File: composer.json [hunk 2]
```diff
@@ -26,20 +26,20 @@
     "require": {
         "php": "^5.6|^7.0",
 
-        "coduo/php-matcher": "^2.0",
-        "doctrine/data-fixtures": "~1.1.0",
-        "doctrine/doctrine-bundle": "^1.6.3",
-        "doctrine/orm": "^2.4.8",
+        "doctrine/orm": "^2.5",
+        "symfony/symfony": "3.1.*",
```
Comment (pamil) [inline composer.json]: `^3.0` should be enough, we should allow for 3.0 and 3.2 too.

Comment (stloyd) [inline composer.json]: Also it would be nice to allow last LTS... `^2.7 || ^3.0`

**Comment by pamil** [comment]
Btw. ApiTestCase is already compatibile with Symfony3, we're using it on Sylius without any issues. Also, requiring the whole `symfony/symfony` package is a massive overhead.

**Comment by siwymilek** [comment]
@pamil Can you have a look at issue #75?

**Comment by pamil** [comment]
@siwymilek will be fixed by #77 :tada:

**Pull request closed without merging**